src/anomalib/utils/cli/subcommands/export.py

```diff
@@ -30,95 +24,41 @@
 def add_torch_export_arguments(subcommand: _ActionSubCommands) -> None:
     """Add torch parser to subcommand."""
     parser = _get_export_parser("torch")
-    subcommand.add_subcommand("torch", parser)
+    parser.add_function_arguments(export_to_torch)
+    subcommand.add_subcommand("torch", parser, help="Export to torch format")
 
 
 def add_onnx_export_arguments(subcommand: _ActionSubCommands) -> None:
     """Add onnx parser to subcommand."""
     parser = _get_export_parser("ONNX")
-    subcommand.add_subcommand("onnx", parser)
+    parser.add_function_arguments(export_to_onnx)
+    subcommand.add_subcommand("onnx", parser, help="Export to ONNX format")
 
 
 def add_openvino_export_arguments(subcommand: _ActionSubCommands) -> None:
     """Add OpenVINO parser to subcommand."""
     if get_common_cli_parser is not None:
         parser = _get_export_parser("OpenVINO")
+        parser.add_function_arguments(export_to_openvino, skip={"mo_args"})
```

Comment:
I guess this is what Adrian needs to work on?

Reply:
While I am exposing the model optimizer parameters to the export method, this could use some refactor.

Reply:
I am skipping this argument as I add this arguments under 
```yaml
mo_args:
    - mo_param1:
    - ...
```
So the type check by jsonargparse fails.